src/customers/account-dashboard-order-by-sku.md

```diff
@@ -3,6 +3,13 @@ ee_only: true
 title: Order by SKU
 ---
 
+SKU stands for Stock Keeping Unit. SKUs help online sellers identify the most important product characteristics such as: size, color, price, material etc. Don't confuse product IDs with SKUs:
```

Comment:
Not all SKUs have a logical convention - this can go back to legacy days.
```suggestion
SKU stands for Stock Keeping Unit. SKUs generally help online sellers identify the most important product characteristics such as: size, color, price, material etc. Don't confuse product IDs with SKUs:
```